pkgs/development/python-modules/jc/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi,
+  ruamel_yaml, ifconfig-parser, xmltodict }:
+
+buildPythonPackage rec {
+  pname = "jc";
+  version = "1.7.2";
```

Comment:
```suggestion
  version = "1.7.2";
  disabled = isPy27;
```

```
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
builder for '/nix/store/xy04rb7vh0k0idggcdj1rbvfgdsky8ky-python2.7-jc-1.7.2.drv' failed with exit code 1; last 10 log lines:
  adding 'jc-1.7.2.dist-info/top_level.txt'
  adding 'jc-1.7.2.dist-info/RECORD'
  removing build/bdist.linux-x86_64/wheel
  Finished executing setuptoolsBuildPhase
  installing
  Executing pipInstallPhase
  /build/jc-1.7.2/dist /build/jc-1.7.2
  DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
  Processing ./jc-1.7.2-py2-none-any.whl
  ERROR: Package 'jc' requires a different Python: 2.7.17 not in '>=3.6'
cannot build derivation '/nix/store/9pnpxixng90alfndkvmc48ghpl47lflp-env.drv
```

Reply:
It builds just fine with python 3 on my machine. The derivation is also completely different, here it is: https://send.firefox.com/download/c7c109191ab3e88a/#YQbpYFcEJA7nNfiACpYorg.

I'm on a normal NixOS with sandboxing enabled, you? Also, there's a warning on top of your log about the sandbox, are you sure it's enabled?


The auto-generated wrapper script also runs it with `/nix/store/g20g00saxyssz0ka1xcmcm31bfq6iq31-python3-3.7.6/bin/python3.7`.

Reply:
adding it to `python-modules` will mean that the python2 package will try to build this, please apply my suggestions

Reply:
TIL, thank you!